pkg/recipe/metadata.go

```diff
@@ -72,3 +75,14 @@ func (m *Metadata) Validate() error {
 
 	return nil
 }
+
+func (m *Metadata) Update(re *Recipe, newVer, msg string) error {
```

Comment:
The method should be called `UpdateVersion` since it only modifies the version and changelog